src/commissaire_service/clusterexec/__init__.py

```diff
@@ -43,6 +44,7 @@ def __init__(self, exchange_name, connection_url):
             {'routing_key': 'jobs.clusterexec.*'}
         ]
         super().__init__(exchange_name, connection_url, queue_kwargs)
+        self.storage = StorageClient(self)
```

Comment:
Why not make ``storage`` part of the base ``CommissaireService`` class? Then it's always available to any service by default.